vision_datasets/common/data_manifest.py

```diff
@@ -742,6 +745,13 @@ def create_dataset_manifest(coco_file_path_or_url, data_type, container_sas_or_r
             images = [x for x in images_by_id.values()]
             return DatasetManifest(images, None, data_type)
 
+        if data_type == DatasetTypes.IMAGE_MATTING:
+            for annotation in coco_manifest['annotations']:
+                label = CocoManifestAdaptor._load_image_matting_label_from_file(get_full_sas_or_path(annotation['label']))
```

Comment:
It is too heavy loading every ground truth into memory:

Maybe add an additional param label_path to ImageDataManifest, where each access to ImageDataManifest.labels will be loading and parsing the labels from file, just like image.

Reply:
To double confirm: do you mean for the ImageDataManifest.labels, we just put the file path to it, and we load the real labels in the vision_benchmark when doing the inference?

Reply:
```
Class ImageDataManifest:
    def __init__(self, ...., labels, label_file_path=None)
         self._labels = labels
         self.label_file_path = label_file_path
    @property
    def labels(self):
        if self._labels:
            return self.labels
        elif self.label_file_path:
            return read_parse(self.label_file_path)
         
         return None
    
```